porch/engine/pkg/engine/init_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package engine
+
+import (
+	"context"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	api "github.com/GoogleContainerTools/kpt/porch/api/porch/v1alpha1"
+	"github.com/GoogleContainerTools/kpt/porch/repository/pkg/repository"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestInit(t *testing.T) {
+	init := &initPackageMutation{
+		name: "testpkg",
+		spec: api.PackageInitTaskSpec{
+			Description: "test package",
+			Keywords:    []string{"test", "kpt", "pkg"},
+			Site:        "http://kpt.dev/testpkg",
+		},
+	}
+
+	testdata, err := filepath.Abs(filepath.Join(".", "testdata", "init", "testpkg"))
+	if err != nil {
+		t.Fatalf("Failed to find testdata: %v", err)
+	}
+
+	rendered, _, err := init.Apply(context.Background(), repository.PackageResources{})
```

Comment:
super nit: the package is not `rendered` so much as `initialized` :)

Reply:
copy paste :) Fixed.